Dockerfile

```diff
@@ -0,0 +1,4 @@
+FROM node:12-slim
```

Comment:
Can you upgrade to Node 14, please?

Reply:
fixed in 4f8d81d971a4ee1142b87a86d8b1fda3c4fa8025